### ISSUE TITLE
Unable to login to FreshRSS with FeedReader

### ISSUE THREAD

**Issue by ggdagg** [jangernert/FeedReader#640]
Hi,

I encounter a similar issue as the one mentioned here #505. I tried to follow some steps suggested in that post but without success.

- I enabled %2F support. Now both full server check and partial server check configuration have a PASS result in FreshRSS.

- I checked that mod_rewrite is correctly enabled on my Apache server as advised:
`[root@example.com bin]# ./apachectl -M`
`Loaded Modules:`
 `[...]`
 `rewrite_module (shared)`

- When I enter https://example.com as URL in FeedReader, I can see the following Apache log with this FeedReader error "Either your username or password are not correct":
` [28/Feb/2018:11:53:25 +0000] "HEAD / HTTP/1.1" 200 - "-" "FeedReader 2.0.2"`
`[28/Feb/2018:11:53:25 +0000] "POST /api/greader.php/accounts/ClientLogin HTTP/1.1" 404 234 "-" "FeedReader 2.0.2"`

- When I enter https://example.com/p/api/greader.php as URL in FeedReader, I can see the following Apache log with this FeedReader error "No connection to the server error. Check your Internet connection and the server URL!":
`[28/Feb/2018:11:54:10 +0000] "HEAD /p/api/greader.php HTTP/1.1" 400 - "-" "FeedReader 2.0.2"`

I don't know where the problem could be (either FeedReader or FreshRSS or Apache) but I tried to set up a second instance of FreshRSS on the same server AND on a different one but I ended up with the same result.
Do you have any idea or suggestion?

Thank you

Server info:
- Apache/2.4.25
- CentOS 7

Client info:
- Fedora 27
- FeedReader 2.0.2 (installed via dnf)

**Comment by brendanlong** [comment]
It looks like your login page is returning 404 - not found:

> [28/Feb/2018:11:53:25 +0000] "POST /api/greader.php/accounts/ClientLogin HTTP/1.1" 404 234 "-" "FeedReader 2.0.2"

I'm not sure what's wrong but it looks like a server configuration issue.

**Comment by ggdagg** [comment]
Thanks for your input.

Instead of using https://example.com, I tried with https://example.com/p/ - I forced the `/p/` at the end of my URL as you noticed the 404 error in my log. However it didn't work.
But I also tried without SSL: http://example.com doesn't work but forcing `/p/` with http://example.com/p/ worked and I was able to log in.

So as you said, it's more likely to be a server configuration issue.
